.github/workflows/pr-labeler.yml

```diff
@@ -16,4 +16,4 @@ jobs:
         ref: main
     - uses: actions/labeler@v5
       with:
-          configuration-path: .github/prs-labels.yml
+          configuration-path: .github/pr-labels.yml
```

Comment:
Is this whitespace correct? It doesn't line up with the above lines at the same indentation (lines 15-16)

Reply:
It works... I YAML isn't too picky about how much you indent sub-objects, just that they are further indented.